docs/creating-a-package.md

```diff
@@ -123,7 +123,7 @@ like you.
 
 Style sheets for your package should be placed in the _stylesheets_ directory.
 Any style sheets in this directory will be loaded and attached to the DOM when
-your package is activated. Style sheets can be written as CSS or [LESS] (but
+your package is activated. Style sheets can be written as CSS or [LESS] \(but
```

Comment:
Just to be clear I was just aiming to fix the technical problem; I don't have an opinion one way or the other on what's actually said there.

I'll go ahead and make the suggested change.